pre-registration/pre-registration-application-service/src/test/resources/application.properties

```diff
@@ -440,7 +441,7 @@ mosip.io.prid.url=${kernel.prid.env}/${masterdata.service.version}/${kernel.prid
 # ----------- Document -------------
 
 # masterdata url to get doctype and catcode
-mosip.kernel.masterdata.validdoc.rest.uri=${masterdata.resource.url}/validdocuments/{langcode}
+mosip.kernel.masterdata.validdoc.rest.uri=${masterdata.resource.url}/validdocuments/all?pageSize=50&pageNumber=0
```

Comment:
why is this change done?

Reply:
yes` mosip.kernel.masterdata.validdoc.rest.uri=${masterdata.resource.url}/validdocuments/{langcode}`  
this `{langcode}`  is creating problem. I tried giving langcode=eng and ara also but  it's redirected to this url  https://dev.mosip.net/v1/masterdata/validdocuments/%7langcode    which is bad request. So after changing to mosip.kernel.masterdata.validdoc.rest.uri=${masterdata.resource.url}/validdocuments/all?pageSize=50&pageNumber=0  this is working fine. I have run all the test cases also from root folder all test cases are running fine.